t/bad_subclass.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
+
+use lib 'lib', 't/lib';
+
+use Test2::V0;
+
+# This test verifies the error handling when a TCM class extends a
+# non-TCM subclass.  All this happens at compile time, so the usual
+# exception testing is not applicable.  The modules get a directory of
+# their own because they (intentionally) break TCM::Load.
+
+use lib 't/badlib';
+
+my $error;
+
+# Verify that the error isn't triggered for correct subclassing
+eval <<EOE;
```

Comment:
Always use some sort of quote with heredocs. In this case I think you can do `eval <<'EOE';`